redpanda/examples/python/kafka-produce-avro.py

```diff
@@ -10,6 +10,7 @@
 # $ cd confluent-kafka
 # $ source bin/activate
 # $ pip3 install confluent-kafka
+# $ pip3 install avro
```

Comment:
Same comment as above, would it make sense to make this part of the note for mac below?